stable/redis-ha/Chart.yaml

```diff
@@ -6,8 +6,8 @@ keywords:
 - redis
 - keyvalue
 - database
-version: 3.6.0
-appVersion: 5.0.3
+version: 3.7.0
```

Comment:
@arbourd I think because it's a patch release of Redis, you could just bump the patch version of the chart as well.

Reply:
Sounds good!